Userland/Libraries/LibWeb/HTML/NavigableContainer.cpp

```diff
@@ -262,25 +262,27 @@ void NavigableContainer::destroy_the_child_navigable()
     // 3. Set container's content navigable to null.
     m_content_navigable = nullptr;
 
-    // 4. Destroy navigable's active document.
+    // FIXME: 4. Inform the navigation API about child navigable destruction given navigable.
+
+    // 5. Destroy navigable's active document.
     navigable->active_document()->destroy();
 
-    // 5. Let parentDocState be container's node navigable's active session history entry's document state.
-    auto parent_doc_state = this->navigable()->active_session_history_entry()->document_state;
+    // 6. Let parentDocState be container's node navigable's active session history entry's document state.
+    auto parent_doc_state = navigable->active_session_history_entry()->document_state;
```

Comment:
it should be `this->navigable()` to access container's node navigable instead of content navigable.